cdap-master/src/main/java/co/cask/cdap/data/tools/QueueConfigUpgrader.java

```diff
@@ -61,7 +61,13 @@ protected TableId getTableId() {
 
   @Nullable
   private QueueName fromRowKey(byte[] oldRowKey) {
-    QueueName queueName = QueueName.from(oldRowKey);
+    QueueName queueName;
+    try {
+      queueName = QueueName.from(oldRowKey);
```

Comment:
Right, but if half the rows get converted, and then the tool is killed, the next time it runs, it would error, right?